Makefile

```diff
@@ -530,28 +522,13 @@ $(TELEKUBE_OUT): packages
 		--skip-version-check \
 		-o $(TELEKUBE_OUT)
 
-$(GRAVITY_OUT):
```

Comment:
I removed the code I added in my previous custom robotest image PR:

https://github.com/gravitational/gravity/pull/1967/files#diff-b67911656ef5d18c4ae36cb6741b7965R533-R537

Is your use of `make gravity tele` more recent than that?  I can definitely add these back in.

I added this code because robotest targets depend on $(GRAVITY_OUT) and $(TELE_OUT).  They used to explicitly do so, but this was unpleasant because these tele and gravity depend on phony targets... meaning they constantly trigger a full rebuild.  For the new `robotest-run-suite` and `robotest-run-nightly`, in this PR I implicitly declare the dependencies in a comment to avoid this.  Maybe these should be an "ordering dependency"  Its not great... but I also don't want to fix our entire dependency tree in this PR -- doing so would be a lot of work and orthogonal to the changes here.

Reply:
This shouldn't affect `make gravity tele`, that uses the `$(BINARIES)` target.  `$(GRAVITY_OUT)` and `$(TELE_OUT)` are the full paths to the binaries, not the .PHONY targets you're calling.